[HWORKS-244] Documentation for exporting Prometheus metrics (#141)
logicalclocks/logicalclocks.github.io: docs/admin/monitoring/export-metrics.md

# Exporting Hopsworks metrics

## Introduction
Hopsworks services produce metrics which are centrally gathered by [Prometheus](https://prometheus.io/) and visualized in [Grafana](../grafana).

> Although the system is self-contained it is possible to export metrics gathered by Prometheus to third-party services or another Prometheus instance.

Reviewer: ```suggestion
Although the system is self-contained, it is possible to export these metrics to third-party services or another Prometheus instance.
```
